pkg/kubectl/cmd/cmd_test.go

```diff
@@ -477,6 +477,8 @@ func Example_printPodHideTerminated() {
 	// NAME      READY     STATUS    RESTARTS   AGE
 	// test1     1/2       Pending   6          10y
 	// test2     1/2       Running   6         10y
+	// test3     1/2       Succeeded   6         10y
```

Comment:
Why CI couldn't find this.

Reply:
> Why CI couldn't find this?


@mikedanese bazel doesn't run examples maybe?